operator/internal/lifecycle/client.go

```diff
@@ -0,0 +1,387 @@
+// Copyright 2025 Redpanda Data, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+package lifecycle
+
+import (
+	"context"
+	"errors"
```

Comment:
This should be cockroachdb/errors. A linter should be yelling at you 🤔 Is it not?

Reply:
Huh... not sure why this isn't triggering an error. Guess that's on me 😓